db/fixtures/miq_policy_sets.yml

```diff
@@ -39,6 +39,21 @@
           description: Mark as Non-Compliant
           action_type: default
           options: {}
+      - qualifier: failure
+        failure_sequence: 2
+        failure_synchronous: true
+        MiqEventDefinition:
+          name: containerimage_compliance_check
```

Comment:
I noticed this as well.  The events are defined in [db/fixtures/miq_event_definitions.csv#L170](https://github.com/ManageIQ/manageiq/blob/db30bc9cf72689956b143d51838acf5c2af5ef4c/db/fixtures/miq_event_definitions.csv#L170).

@jrafanie Seems both of us missed it when reviewing PR #7686 here https://github.com/ManageIQ/manageiq/pull/7686/files#r58567191

@moolitayer Can you identify why these event names were originally set to `containerimage_*` and look into correcting that.